internal/util/importutil/import_wrapper.go

```diff
@@ -186,7 +186,7 @@ func (p *ImportWrapper) fileValidation(filePaths []string, rowBased bool) error
 		size, err := p.chunkManager.Size(context.TODO(), filePath)
 		if err != nil {
 			log.Error("import wrapper: failed to get file size", zap.String("filePath", filePath), zap.Any("err", err))
-			return errors.New("failed to ")
+			return errors.New("failed to get file size of " + filePath)
```

Comment:
Prefer fmt.Errorf() instead.

Reply:
There are several places use errors.New(). Change all of them can conflict with #19671. I will refine them in the pr #19671